velodyne_driver/src/driver/driver.cc

```diff
@@ -169,7 +169,7 @@ VelodyneDriver::VelodyneDriver(ros::NodeHandle node,
   diag_topic_.reset(new TopicDiagnostic("velodyne_packets", diagnostics_,
                                         FrequencyStatusParam(&diag_min_freq_,
                                                              &diag_max_freq_,
-                                                             0.1, 10),
```

Comment:
Not sure it is a dynamic reconfigure now?

Reply:
why not? on the right I use config_.frequency_tolerance

Reply:
Yes indeed, but we are in the class constructor here, so this only gets called on start. Hence I don't think later reconfigure changes will be noticed by `FrequencyStatusParam()` or `TopicDiagnostic()`

However, I think making it a parameter is already an improvement, reconfigure could be nice to have?

Reply:
Ahh yes I had missed that one... I agree for now a parameter is enough. I will make the change